lua/telescope/builtin/internal.lua

```diff
@@ -403,27 +403,33 @@ internal.help_tags = function(opts)
 end
 
 internal.man_pages = function(opts)
-  local pages = utils.get_os_command_output(opts.man_cmd or { 'apropos', '--sections=1', '' })
+  opts.sections = utils.get_default(opts.sections, {'1'})
+  assert(vim.tbl_islist(opts.sections), 'sections should be a list')
+  opts.man_cmd = utils.get_lazy_default(opts.man_cmd, function()
+    return vim.fn.has'mac' and {'apropos', ' '} or {'apropos', ''}
+  end)
+  opts.entry_maker = opts.entry_maker or make_entry.gen_from_apropos(opts)
+  opts.PAGER = utils.get_lazy_default(opts.PAGER, function()
+    return vim.fn.executable('col') and 'col -bx' or ''
+  end)
```

Comment:
I wanted that in previewers. No need to pass it in a opts. You can do that.
```lua
previewers.man = defaulter(function()
  local pager = ... -- this will run once. When the previewer gets created
  return previewers.new_buffer_previewer {
```

Reply:
Got it. Moved logic but still can select `PAGER` by `opts.PAGER`. a8ee358